### PR TITLE
Delete unused flowcell_samples page, add css rules, update Cronjobs page to bootstrap5

### PR DESCRIPTION
**Pull request by aanil**
Up on stage!

### CODE REVIEW

**Review by sofiahaglund — APPROVED** [review]
Oh I had forgotten about the flowcell samples page :-) 

I can't really comment about the cronjobs, but maybe Johannes has more insight?

Visually everything looks great though!

File: run_dir/design/cronjobs.html [hunk 1]
```diff
@@ -1,28 +1,30 @@
...
   {% for server, crontab in sorted(cronjobs.items(), key=lambda x:x[1].get('last_updated', ''), reverse=True) %}
     {% set two_month_ago=str(datetime.datetime.now() - datetime.timedelta(30*2))%}
-    <div class="panel {% if crontab.get('last_updated', '') < two_month_ago %} panel-warning {% else %} panel-default {% end %}">
-      <div class="panel-heading">
-        <h4 class="panel-title">
-          <a data-toggle="collapse" data-parent="#cronjobs_accordion" href="#cronjobs_{{ server }}">
+    <div class="accordion-item accordion-item-prettify">
+			<div class="accordion-default-header {% if crontab.get('last_updated', '') < two_month_ago %} accordion-warning {% end %}">
+        <h4 class="accordion-header">
+          <a class="text-decoration-none {% if crontab.get('last_updated', '') < two_month_ago %} text-warning{% else %} text-dark{% end %} collapsed"
```
Comment (alneberg) [inline run_dir/design/cronjobs.html:15]: I think it's enough with the warning color on the header background and keep the text-dark for all.

File: run_dir/design/cronjobs.html [hunk 2]
```diff
@@ -1,28 +1,30 @@
...
-      <div id="cronjobs_{{ server }}" class="panel-collapse collapse">
-        <div class="panel-body">
+			</div>
+      <div id="cronjobs_{{ server }}" class="accordion-collapse collapse collapsed"
+					data-parent="#cronjobs_accordion" {% if crontab.get('last_updated', '') < two_month_ago %} style="border-color: #fff3cd;" {% end %} >
+        <div class="accordion-body">
           <table class="table">
             <thead>
-              <tr>
+              <tr class="darkth">
```
Comment (alneberg) [inline run_dir/design/cronjobs.html:27]: I prefer it without the `.darkth` on these tables.

Comment (aanil) [reply run_dir/design/cronjobs.html]: Done!

File: run_dir/design/cronjobs.html (hunk 1 above)
Comment (aanil) [reply run_dir/design/cronjobs.html]: Done!

**Review by alneberg — APPROVED** [review]

**Pull request merged**